src/extensions/mvs/tree/mvs/mvs-tree-primitives.ts

```diff
@@ -75,6 +75,39 @@ const TubeParams = {
     tooltip: OptionalField(nullable(str), null, 'Tooltip to show when hovering over the tube. If not specified, uses the parent primitives group `tooltip`.'),
 };
 
+const ArrowParams = {
+    /** Start point of the tube. */
+    start: RequiredField(PrimitivePositionT, 'Start point of the arrow.'),
+    /** End point of the tube. */
+    end: OptionalField(nullable(PrimitivePositionT), null, 'End point of the arrow.'),
+    /** If specified, the endpoint is computed as start + direction. */
+    direction: OptionalField(nullable(Vector3), null, 'If specified, the endpoint is computed as start + direction.'),
+    /** Length of the arrow. If unset, the distance between start and end is used. */
+    length: OptionalField(nullable(float), null, 'Length of the arrow. If unset, the distance between start and end is used.'),
+    /** Draw an arrow at the start of the arrow. */
+    arrow_start: OptionalField(bool, false, 'Draw an arrow at the start of the arrow.'),
+    /** Height of the arrow at the start. */
+    arrow_start_height: OptionalField(float, 0.1, 'Height of the arrow at the start.'),
+    /** Radius of the arrow at the start. */
+    arrow_start_radius: OptionalField(float, 0.1, ''),
+    /** Draw an arrow at the end of the arrow. */
+    arrow_end: OptionalField(bool, false, 'Draw an arrow at the end of the arrow.'),
+    /** Height of the arrow at the end. */
+    arrow_end_height: OptionalField(float, 0.1, 'Height of the arrow at the end.'),
```

Comment:
I'm not 100% sure what arrow_start_height, arrow_start_radius, arrow_end_height, arrow_end_radius mean, but if wouldn't it make more sense to call it *_length instead of *_height?

And also use term "arrow cap" (or something similar) to distinguish it from "arrow" as a whole (at least in descriptions, as in param names it might be too wordy).